### PR TITLE
HTML formatting

### PR DESCRIPTION
**Pull request by maciaszczykm**
Added HTML code formatter as it was discussed in #371 issue. 

Unfortunately it has problems with keeping max line length at exact number, rest of the settings seems to be okay.

[Review on Reviewable](https://reviewable.io/reviews/kubernetes/dashboard/377)

### CODE REVIEW

File: src/app/frontend/logs/logstoolbar/logstoolbar.html [hunk 1]
```diff
@@ -39,15 +35,13 @@
     <span ng-if="ctrl.podCreationTime">
       Running since {{ctrl.podCreationTime | date:'short'}}
     </span>
-    <span ng-if="!ctrl.podCreationTime">
+  <span ng-if="!ctrl.podCreationTime">
```
Comment (bryk) [inline src/app/frontend/logs/logstoolbar/logstoolbar.html]: Is this correct?

File: src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail.html [hunk 2]
```diff
@@ -33,241 +30,220 @@ <h1 flex class="md-title kd-replicationcontrollerdetail-app-name">
...
+    <div class="kd-replicationcontrollerdetail-sidebar-header-wrapper kd-replicationcontrollerdetail-sidebar-item"
+      flex="nogrow" ng-show="!ctrl.isSidebarVisible()">
+      <div class="kd-replicationcontrollerdetail-sidebar-header" layout-align="start center" layout="row">
+        <md-button class="md-icon-button" ui-sref="replicationcontrollers">
+          <md-icon md-font-library="material-icons">arrow_back</md-icon>
+        </md-button>
+        <h1 flex class="md-title kd-replicationcontrollerdetail-app-name">
         {{::ctrl.replicationControllerDetail.name}}
       </h1>
+      </div>
```
Comment (bryk) [inline src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail.html]: Is this correct?

**Comment by bryk** [comment]
I don't know... The output looks wrong for many `span`s. What do you think?

File: src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail.html (hunk 2 above)
Comment (maciaszczykm) [inline src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail.html]: `<div>` content should be indented. `{{::ctrl.replicationControllerDetail.name}}` and `</h1>` are not. It's wrong.

File: src/app/frontend/logs/logstoolbar/logstoolbar.html (hunk 1 above)
Comment (maciaszczykm) [inline src/app/frontend/logs/logstoolbar/logstoolbar.html]: No...

**Comment by maciaszczykm** [comment]
@bryk I agree, it's how `gulp-prettify` is working.

**Comment by floreks** [comment]
How about trying to use this instead?

http://www.html-tidy.org/

It doesn't have direct gulp plugin but we should be able to execute its binary file with gulp.

**Comment by bryk** [comment]
How about you ignore this for now and take a look at bugs with milestone=v1.0?

**Comment by maciaszczykm** [comment]
@bryk Sure, no problem.

**Comment by bryk** [comment]
Closing as stale. Let's reopen if we think this is important

**Pull request closed without merging**